src/test/java/org/broadinstitute/hellbender/utils/io/IOUtilsUnitTest.java

```diff
@@ -41,6 +44,212 @@ public void tearDown() {
         }
     }
 
+    private void assertContentsTheSame(final Path baseActualPath, final Path baseExpectedPath) {
+        try {
+            // Check that the files and directories are the same:
+            Files.find(baseActualPath, Integer.MAX_VALUE,
```

Comment:
Problem: if `baseActualPath` is missing one or more files that are present in `baseExpectedPath` (or if `baseActualPath` is completely empty), this method won't throw an error, since the code in the lambda will never get invoked for those missing files. You need either a counter for actual->expected matches, or a symmetric check.

Reply:
Fixed!